app.yaml

```diff
@@ -48,8 +49,7 @@ handlers:
   application_readable: true
   http_headers:
     Cache-Control: 'public, max-age=60'
-- url: /css
-  # NB: not minified. TODO(sll): fix.
+- url: /templates/dev/head/css
```

Comment:
The rule below is for js files, but it would be probably better to extend it to capture CSS files too then to have two separate rules.